btree/btree.c

```diff
@@ -4677,6 +4760,9 @@ static int64_t btree_del_resolve_underflow(struct m0_btree_op *bop)
 	lev->l_node->n_skip_rec_count_check = false;
 	oi->i_level[1].l_sibling->n_skip_rec_count_check = false;
 
+	n_lock_op_unlock(lev->l_node);
```

Comment:
This unlock might be extra.
Please check once.

Reply:
It is not the extra unlock, At line no. 4708, if used_count ==0 and node_count_rec(lev->l_node) == 1, we will break without unlock.
This is the case when we have reach root node and there is only one record in root so we want to copy that record in root node so that we can decrease the height of tree

Reply:
Oh Yes.
I see that.
Thanks.